src/main/java/com/saucelabs/jenkins/pipeline/ExpanderImpl.java

```diff
@@ -3,7 +3,7 @@
 import hudson.EnvVars;
 import org.jenkinsci.plugins.workflow.steps.EnvironmentExpander;
 
-import javax.annotation.Nonnull;
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
Matching our preference to migrate away from JSR 305 throughout the plugin ecosystem.